contracts/token/ERC721/IERC721.sol

```diff
@@ -0,0 +1,40 @@
+pragma solidity ^0.4.24;
+
+import "./IERC721Basic.sol";
+
+
+/**
+ * @title ERC-721 Non-Fungible Token Standard, optional enumeration extension
+ * @dev See https://github.com/ethereum/EIPs/blob/master/EIPS/eip-721.md
+ */
+contract IERC721Enumerable is IERC721Basic {
```

Comment:
I forget why, but please refresh my memory; since these are `contract`s, they can hold storage variables, so why are the `InterfaceId_ERC721Enumerable` variables (which should be `InterfaceId_IERC721Enumerable`?) not in this interface and are instead in the implementation (ERC721Basic)?

Reply:
I think we just forgot to move them.